docs/concepts/ClusterResourcePlacement/README.md

```diff
@@ -0,0 +1,235 @@
+that resources can change independently of any modifications to the `ClusterResourceSnapshot`. In other words, resource
+changes can occur without directly affecting the `ClusterResourceSnapshot` itself.
+
+The total amount of selected resources may exceed the 1MB limit for a single Kubernetes object. As a result, the controller 
+may produce more than one `ClusterResourceSnapshot`s for all the selected resources.
+
+## Placement Policy
+
+`ClusterResourcePlacement` supports three types of policy as mentioned above. `ClusterSchedulingPolicySnapshot` will be
+generated whenever policy changes are made to the `ClusterResourcePlacement` that require a new scheduling. Similar to
+`ClusterResourceSnapshot`, its spec is immutable.
+
+![](scheduling.jpg)
+
+Compared with the Kubernetes' original scheduler framework, the multi-cluster scheduling selects a cluster for the placement 
+in a 5-step operations:
+1. Batch & PostBatch
+2. Filter 
+3. Score
+4. Sort
+5. Bind
+
+The _batch & postBatch_ step is to define the batch size according to the desired and current `ClusterResourceBinding`. 
+The postBatch is to adjust the batch size if needed.
+
+The _filter_ step finds the set of clusters where it's feasible to schedule the placement, for example, whether the cluster
+is matching required `Affinity` scheduling rules specified in the `Policy`. It also filters out any clusters which are 
+leaving the fleet or no longer connected to the fleet, for example, its heartbeat has been stopped for a prolonged period of time.
+
+In the _score_ step (only applied to the pickN type), the scheduler assigns a score to each cluster that survived filtering.
+Each cluster is given a topology spread score (how much a cluster would satisfy the topology spread
+constraints specified by the user), and an affinity score (how much a cluster would satisfy the preferred affinity terms
+specified by the user). 
+
+In the _sort_ step (only applied to the pickN type), it sorts all eligible clusters by their scores, sorting first by topology 
+spread score and breaking ties based on the affinity score.
+
+The _bind_ step is to create/update/delete the `ClusterResourceBinding` based on the desired and current member cluster list.
+
+## Rollout Strategy
+Update strategy determines how changes to the `ClusterWorkloadPlacement` will be rolled out across member clusters. 
+The only supported update strategy is `RollingUpdate` and it replaces the old placed resource using rolling update, i.e. 
+gradually create the new one while replace the old ones.
+
+## Placement status
+
+After a `ClusterResourcePlacement` is created, details on current status can be seen by performing a `kubectl describe crp <name>`.
+The status output will indicate both placement conditions and individual placement statuses on each member cluster that was selected.
+The list of resources that are selected for placement will also be included in the describe output. 

```

Comment:
can we add links to the deep dive  "how to" section of the doc here after they are done?